charmander/routes/order/controller.py

```diff
@@ -19,3 +20,18 @@ def __init__(self, graph):
             subject=Order,
             version="v1",
         )
+        self.sns_producer = graph.sns_producer
+        self.order_event_factory = graph.order_event_factory
+
+    def create(self, **kwargs):
+        order = super().create(**kwargs)
+
+        self.order_event_factory.create(
```

Comment:
When other resources in `charmander` need to create an `order_event` they do so through the factory and do not have to worry about any special logic or cases associated with creating different types of events.